[IMP] helpdesk: update ratings doc (#3664)
odoo/documentation: content/applications/services/helpdesk/overview/ratings.rst

================
Customer ratings
================

Asking customers to rate their experience with :guilabel:`Helpdesk` support provides an opportunity
to gauge team performance, and track customer satisfaction. Ratings can published to the portal,
providing customers with a general overview of the team's performance.

Enable customer ratings on a Helpdesk team
==========================================

To enable :guilabel:`Customer Ratings` on a Helpdesk team, go to :menuselection:`Helpdesk -->
Configuration --> Teams`. Select a team from the list and navigate to the settings page. Scroll to
the :guilabel:`Performance` section, and check the box for :guilabel:`Customer Ratings`.

.. image:: ratings/ratings-enable.png
   :align: center
   :alt: Overview of the settings page of a helpdesk team emphasizing the rating on ticket feature
         in Odoo Helpdesk

Set a ratings request email template on a stage
===============================================

To automatically request ratings from customers once their tickets have closed, an email template
needs to be added to the appropriate stage.

> To choose a stage, go to :menuselection:`Configuration --> Stages`. Or, from a team's settings

Author: Thank you for catching this. I am removing the menu "configuration -> stages" reference, just to avoid any unnecessary confusion/updates. It actually makes more sense to direct users to the stage list from the settings page, because they will theoretically be there anyway if there are following the steps.

> page, click :guilabel:`Set an Email Template on Stages` under :guilabel:`Customer Ratings`. Select

Reviewer: ```suggestion
page, click :guilabel:`Set an Email Template on Stages` under :guilabel:`Customer Ratings`. Once the correct stage is selected, navigate to the :guilabel:`Email Template`, and add the appropriate template for requesting customer ratings: `Helpdesk: Ticket Rating Request`.
```

one of the stages from the list to add a ratings request template.

.. important::
   Customers should only be asked to rate tickets once an issue has been resolved and their ticket
   is closed. Therefore, a *ratings request* email should only be added to a stage that is

>    :guilabel:`Folded in Kanban`, as tickets in a *folded stage* are considered closed.

Reviewer: Maybe note where this can bee seen on the stage?

> On the stage's settings page, select `Helpdesk: Ticket Rating Request` in the :guilabel:`Email

Reviewer: ```suggestion

```

> Template` field. This template has been pre-configured with ratings customers can use to provide

Reviewer: ```suggestion
The Helpdesk: Ticket Rating Request` template has been pre-configured with ratings customers can use to provide
```

feedback. To view the template, click the arrow button to the right of the field.

Once the template has been added to the stage, it will automatically send a message when a ticket is
moved to that stage. Customers will be asked to rate the support they received with colored icons.

>  - *Green smiling face* - Satisfied

Reviewer: I would use the example block for these face descriptions https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#example

 - *Yellow neutral face* - Okay
 - *Red frowning face* - Dissatisfied

.. image:: ratings/ratings-customer-email.png
   :align: center
   :alt: View of a standard helpdesk customer review email template for Odoo Helpdesk

After selecting a rating, customers are taken to a webpage where they can provide specific written
feedback to support their rating. Once a rating is submitted, it is added to the chatter on the
ticket.

.. tip::
   Customer ratings can also be viewed through the :guilabel:`Customer Ratings` report. To view
   this report, go to :menuselection:`Helpdesk --> Reporting --> Customer Ratings`.

.. seealso::
   - :doc:`/applications/general/email_communication/email_template`

Publish ratings on the customer portal
======================================

After enabling the :guilabel:`Customer Ratings` setting, an option to publish ratings on the team's
website appears. Enabling this setting provides portal users with an overview of the ratings the
team has received over the last thirty days. Specific written feedback will not be included, only

> statistics of the team's performance.

Reviewer: Is this public facing? Ie customers?

.. important::

>    In order to display ratings on the customer portal, a team has to be be visible to

Reviewer: ```suggestion
   In order to display ratings on the customer portal, a team has to be be visible or set to
```

>    :guilabel:`Invited portal users and all internal users`. This setting is configured on the

Reviewer: Needs to be reword or guilabel should be removed

   team's settings page under :guilabel:`Visibility`.

> Go to :menuselection:`Helpdesk --> Configuration --> Teams` and select a team. Scroll to

Reviewer: ```suggestion
Next, to publish the ratings, go to :menuselection:`Helpdesk --> Configuration --> Teams` and select a team. Scroll to
```

:guilabel:`Performance` and enable :guilabel:`Publish this team's ratings on your website`.

To view the ratings for a team, a customer will log into the portal and navigate to one of their
tickets. After clicking on the team name in the :guilabel:`Managed By` field, they will be directed
to a page with the team's ratings over the past thirty days.

.. image:: ratings/ratings-portal-overview.png
   :align: center

>    :alt: View of the ratings performance overview from the customer portal

Reviewer: ```suggestion
   :alt: View of the ratings performance overview from the customer portal. 
```

Should have periods at the end of an Alt tag.

Manually hide individual ratings
--------------------------------

Individual ratings can be manually hidden from the portal. This allows for specific ratings to be
kept out of the performance metrics that are shown to customers.

To make a rating visible only to internal users, navigate to the page for a rating. This can be done
in one of the following ways:

>    - Go to the page for a rated ticket and click the :guilabel:`Rating` smart button.

Reviewer: How do I get to the page?

================
Customer ratings
================

> Asking customers to rate the support they received from a Helpdesk team provides an opportunity to

Reviewer 2: Lets italicize *Helpdesk Teams* since we are introducing the concept/term

> Asking customers to rate the support they received from a *Helpdesk Team* provides an opportunity to

Reviewer 3: ```suggestion
Asking customers to rate the support they received from a *Helpdesk* team provides an opportunity to
```

gauge team performance and track customer satisfaction. Ratings can be published to the portal,
providing customers with a general overview of the team's performance.

Enable customer ratings on a helpdesk team
==========================================

> 

Reviewer 2: lowercase helpdesk

> To enable :guilabel:`Customer Ratings` on a helpdesk team, go to :menuselection:`Helpdesk -->

Reviewer 3: ```suggestion
To enable *customer ratings* on a helpdesk team, go to :menuselection:`Helpdesk -->
```

Configuration --> Teams`. Select a team from the list and navigate to the settings page. Scroll to
the :guilabel:`Performance` section, and check the box for :guilabel:`Customer Ratings`.

.. image:: ratings/ratings-enable.png
   :align: center
   :alt: Overview of the settings page of a helpdesk team emphasizing the rating on ticket feature
         in Odoo Helpdesk.

Set a ratings request email template on a stage
===============================================

To automatically request ratings from customers once their tickets have closed, an email template
should be added to the appropriate stage.

> Once :guilabel:`Customer Ratings` has been enabled on the team's settings page, (see above) click
> the :guilabel:`Set an Email Template on Stages` link. Select a stage from the list, or click
> :guilabel:`New` to create a new stage.

Reviewer 3: Justify the guilabel
```suggestion
Once the :guilabel:`Customer Ratings` setting has been enabled on the team's settings page, (see
above) click the :guilabel:`Set an Email Template on Stages` link. Select a stage from the list, or
click :guilabel:`New` to create a new stage.
```

.. important::
   Customers should only be asked to rate tickets once an issue has been resolved and their ticket
   is closed. Therefore, a *ratings request* email should only be added to a stage that is

>    :guilabel:`Folded in Kanban`, as tickets in a *folded stage* are considered closed.

Reviewer 3: No guilabel here.
Instead would bolden it to bring attention and then emphasize again later in the sentence.
```suggestion
   **folded** in the kanban, as tickets in a *folded stage* are considered closed.
```

On the stage's settings page, select `Helpdesk: Ticket Rating Request` in the :guilabel:`Email
Template` field. This template has been pre-configured with ratings customers can use to provide
feedback. To view the template, click the arrow button to the right of the field.

Once the template has been added to the stage, it will automatically send a message when a ticket is
moved to that stage. Customers will be asked to rate the support they received with colored icons.

 - *Green smiling face* - Satisfied
 - *Yellow neutral face* - Okay
 - *Red frowning face* - Dissatisfied

.. image:: ratings/ratings-customer-email.png
   :align: center

>    :alt: View of a standard helpdesk customer review email template for Odoo Helpdesk

Reviewer 2: add period at end of alt tag

After selecting a rating, customers are taken to a webpage where they can provide specific written
feedback to support their rating. Once a rating is submitted, it is added to the chatter on the
ticket.

.. tip::
   Customer ratings can also be viewed through the :guilabel:`Customer Ratings` report. To view
   this report, go to :menuselection:`Helpdesk --> Reporting --> Customer Ratings`.

.. seealso::

>    - :doc:`/applications/general/email_communication/email_template`

Reviewer 2: remove bullet point from a single seealso link

Publish ratings on the customer portal
======================================

After enabling the :guilabel:`Customer Ratings` setting, an option to publish ratings on the team's
website appears. Enabling this setting provides portal users with an overview of the ratings the

> team has received over the last thirty days. Specific written feedback will not be included, only
> statistics of the team's performance.

Reviewer 3: Specificity + semicolon works here
```suggestion
team has received over the last thirty days. Specific written feedback will not be included; only
statistics of the team's performance will be visible.
```

.. important::

>    In order to display ratings on the customer portal, a team has to have their visibility set to
>    :guilabel:`Invited portal users and all internal users`. This setting is found on the team's
>    settings page under :guilabel:`Visibility`.

Reviewer 3: Justify the guilabel
```suggestion
   In order to display ratings on the customer portal, a team has to have their visibility setting
   set to :guilabel:`Invited portal users and all internal users`. This setting is found on the
   team's settings page under :guilabel:`Visibility`.
```

Next, to publish the ratings, go to :menuselection:`Helpdesk --> Configuration --> Teams` and
select a team. Scroll to :guilabel:`Performance` and enable :guilabel:`Publish this team's ratings
on your website`.

To view the ratings for a team, a customer will log into the portal and navigate to one of their
tickets. After clicking on the team name in the :guilabel:`Managed By` field, they will be directed
to a page with the team's ratings over the past thirty days.

.. image:: ratings/ratings-portal-overview.png
   :align: center
   :alt: View of the ratings performance overview from the customer portal.

Manually hide individual ratings
--------------------------------

Individual ratings can be manually hidden from the portal. This allows for specific ratings to be
kept out of the performance metrics that are shown to customers.

To make a rating visible only to internal users, navigate to the page for a rating. This can be done
in one of the following ways:

   - Go to :menuselection:`Helpdesk --> Reporting --> Customer Ratings` and click on one of the
     kanban cards for an individual rating.

>    - :menuselection:`Tickets --> All Tickets` and remove the :guilabel:`Open` filter from the search
>      bar. Then filter by :guilabel:`Satisfied`, :guilabel:`Okay` and/or :guilabel:`Dissatisfied`.
>      Select a ticket from the results. Click the :guilabel:`Rating` smart button.

Reviewer 3: Incomplete instruction
```suggestion
   - Navigate to :menuselection:`Helpdesk --> Tickets --> All Tickets` and remove the
     :guilabel:`Open` filter from the search bar. Then filter by :guilabel:`Satisfied`,
     :guilabel:`Okay` and/or :guilabel:`Dissatisfied`. Select a ticket from the results. Click the
     :guilabel:`Rating` smart button.
```

Once on the rating details page, check the :guilabel:`Visible Internally Only` box.

.. image:: ratings/ratings-keep-internal.png
   :align: center
   :alt: View of the ratings performance overview from the customer portal.

.. seealso::
   - :doc:`../advanced/close_tickets`

>    - :doc:`../overview/reports`

Reviewer 2: update references to use global path

providing customers with a statistical overview of the team's performance.

Set up
======

To enable :guilabel:`Customer Ratings`, go to :menuselection:`Helpdesk --> Configuration --> Teams`.
Select a team, scroll to :guilabel:`Performance`, and select :guilabel:`Customer Ratings`.

.. image:: ratings/ratings_enable.png
   :align: center
   :alt: Overview of the settings page of a helpdesk team emphasizing the rating on ticket feature
         in Odoo Helpdesk

To automatically request ratings from customers once a ticket has closed, an email template must
be added to the appropriate stage. Click the :guilabel:`Set an Email Template on Stages` button.
Select a :guilabel:`Stage`.

.. important::
   Customers should only be asked to rate tickets once their issue has been resolved and their ticket
   is closed. Therefore, a *Ratings Request* email should only be added to a stage that is
   :guilabel:`Folded in Kanban`, as tickets in a folded stage are considered closed.

.. image:: ratings/ratings_template_onstage.png
   :align: center
   :alt: Overview of a helpdesk team kanban view emphasizing the menu edit stage in Odoo Helpdesk

> Select an email template, or create a new one. Click :guilabel:`Save & Close`. Now, once a ticket

Reviewer 4: Should there be an underscore between & and Close here or is it a typo?

> reaches the folded stage an email is sent to the customer requesting a rating.

Reviewer 4: Should "folded stage" be `:guilabel:` or italicized?

Author: I think italicized, since it should be emphasized (it is a pretty significant change from 15) but it isn't a direct reference in the UI.
